internal/pkg/keys/encrypter_init.go

```diff
@@ -0,0 +1,147 @@
+package keys
+
+import (
+	"fmt"
+	"github.com/frain-dev/convoy/database"
+	"github.com/frain-dev/convoy/pkg/log"
+	"github.com/jmoiron/sqlx"
+	"strings"
+)
+
+func InitEncryption(db database.Database, km KeyManager, encryptionKey string) error {
```

Comment:
We should pass a log instance to this func so we can respect the currently running process' log level.

Reply:
Ok